src/election.js

```diff
@@ -67,10 +67,10 @@ export default class Election {
     validate() {
         return !(
             this.validElectionUrl(this.electionUrl) &&
-            !isNaN(this.electionNum) &&
-            !isNaN(this.repNominate) &&
-            !isNaN(this.numCandidates) &&
-            !isNaN(this.numPositions) &&
+            !Number.isNaN(this.electionNum) &&
+            !Number.isNaN(this.repNominate) &&
+            !Number.isNaN(this.numCandidates) &&
```

Comment:
This change is reverted in 15745f3 a bit further down the line - ideally, we would want to use the recommended `Number.isNaN` instead of the quarky `isNaN`, but switching to it fails at least one test, so I am reluctant to proceed without further investigation